frontend/src/lib-customizations/espoo/citizen/assets/i18n/en.tsx

```diff
@@ -148,40 +148,40 @@ const en: Translations = {
     backToSearch: 'Back to search'
   },
   calendar: {
-    title: 'Calendar',
-    holiday: 'Holiday',
-    absent: 'Absent',
+    title: 'Kalenteri',
```

Comment:
So what's our plan with i18n during piloting? There surely are some non-Finnish speakers there so I don't think we can release this without at least some level of translations.

Reply:
Didn't we agree to just use Finnish for now? I think it's better to use Finnish only instead of incomplete and possibly misleading translations.